phobos/io/xml_factory.py

```diff
@@ -93,7 +89,7 @@ def to_xml(self, object):
         # children that are created from a simple property and have only attributes
         for tag, attribute_map in self.xml_attribute_children.items():
             _attrib = {attname: self._serialize(getattr(object, varname))
-                       for attname, varname in attribute_map.items() if getattr(object, varname) is not None}
+                                        for attname, varname in attribute_map.items() if getattr(object, varname) is not None}
```

Comment:
This indentation does not comply to PEP8 it's to far indented, please revert it.

Reply:
I am still wondering where this came from